src/pages/display-options.tsx

```diff
@@ -275,6 +275,24 @@ export default function DisplayOptions() {
                             value={siteSettings.wallpaper}
                         />
                     </div>
+                    <div className="bg-primary grid grid-cols-2 gap-2">
+                        <div className="flex items-center gap-1 mb-1">
+                            <span className="text-sm">Cycle background every 5 minutes</span>
```

Comment:
I considered letting the user specify a cycle interval (5m, 15m, 30m) but decided to keep things simple to be more aligned with the existing binary settings.